Vagrantfile

```diff
@@ -0,0 +1,29 @@
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/trusty64"
+
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  config.vm.network :private_network, ip: "192.168.33.10"
+
+  config.vm.synced_folder "./app", "/vagrant/app", id: "vagrant-root",
+    owner: "vagrant",
+    group: "www-data",
+    mount_options: ["dmode=775,fmode=664"]
```

Comment:
Services installed by ansible don't use www-data groups, this permissions are also redundant.